scripts/migrate-ideas.mjs

```
@@ -0,0 +1,77 @@
+date: ${frontmatter.date}
+tags: [${frontmatter.tags.map(t => `'${t}'`).join(', ')}]
+status: ${frontmatter.status}
+externalLinks: [${frontmatter.externalLinks.map(link => `'${link.replace(/'/g, "''")}'`).join(', ')}]
+implementation: '${frontmatter.implementation}'
+summary: '${frontmatter.summary}'
+---
+
+${Description}
+`
+
+    // Write to file
```

ESLint

[error] Delete `··`

(prettier/prettier)